### DISCUSSION TITLE
拦截贴吧广告速度变慢

### DISCUSSION THREAD

**Discussion by extraordinarymango** [gkd-kit/gkd#780] [Q&A]
之前八月份的app拦截贴吧广告速度超快。几乎全部拦截。更到最新版本反而拦截速度变慢，点开帖子的第一个广告不拦截，或者反应一会才会拦截。

**Comment by lisonge** [comment]
需要各个版本日志及其对应规则，最好给个视频展示

你只反馈一句话是无法解决问题的

**Reply by lisonge** [reply]
另外我刚刚试了一下

我在主页-进吧-点击任意贴吧，进入后广告是立马点击的

没有出现你说的情况
